packages/react-core/src/components/Select/SelectGroup.tsx

```diff
@@ -20,11 +20,11 @@ export const SelectGroup: React.FunctionComponent<SelectGroupProps> = ({
   titleId = '',
   ...props
 }: SelectGroupProps) => (
-  <div {...props} className={css(styles.selectMenuGroup, className)}>
```

Comment:
This was fine as it is. The structure should be

```
div.selectMenuGroup
    div.selectMenuGroupTitle
    ul
        li
        li
        ....
```

You can see the basic structure in the favorites example, except the `li`'s in the favorites are `li.pf-c-select__menu-wrapper` and they should be `li.pf-c-select__menu-item` in non-favorites lists.